twitter/archivestream.py

```diff
@@ -19,6 +19,8 @@
 import logging
 import argparse
 import time
+import json
+from tweetarchiver.twitterstream import Stream as TwitterStream
 
 from tweetarchiver.tweetstream import TweetStream
```

Comment:
`TweetStream` is no longer used, right? Let's drop import!

Reply:
Yes, I removed it and pushed an update